bot/exts/info/information.py

```diff
@@ -480,19 +485,29 @@ def format_fields(self, mapping: Mapping[str, Any], field_width: int | None = No
         # remove trailing whitespace
         return out.rstrip()
 
-    async def send_raw_content(self, ctx: Context, message: Message, json: bool = False) -> None:
+    async def send_raw_content(self, ctx: Context | Interaction, message: Message, json: bool = False) -> None:
```

Comment:
I think this function should be simplified into just a function that accepts a message, and returns a `str` or `list[str]` (to accommodate for pagination). Accepting both a Context and Interaction feels a bit janky in my opinion.

Reply:
I didn't feel quite the same way at first, but due to other reasons, this route of implementation is frankly *garbage.* I'll be moving the message sending responsibilities to the... overlying(?) methods.

Reply:
> I'll be moving the message sending responsibilities to the... overlying(?) methods.
That sounds like a good idea, now that we have two different methods that send the message in two different ways (normal `!raw` command and context-menu command. Former needs to use `ctx.send` while the latter needs to use `interaction.response.send_message`). It would make sense to practice principle of least responsibility here and have one function that's just responsible for fetching the raw content.

Reply:
I've pushed some new changes that made `send_raw_content` only fetch and return a `Paginator` object instead. Feel free to take a look at the new commits while I go fix the tests. I think they're borked with the `ContextMenu` being there now.

Reply:
Er, take that back. Tests passed with flying colors, haha!